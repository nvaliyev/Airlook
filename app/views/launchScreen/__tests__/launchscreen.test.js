import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { LaunchScreen } from '..';
describe('Laucnhscreen', () => {
    test('Launchscreensnapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<LaunchScreen/>);
        expect(snapshot).toMatchSnapshot();
    });
});


