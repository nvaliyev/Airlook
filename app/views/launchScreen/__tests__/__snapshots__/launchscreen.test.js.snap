// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Laucnhscreen Launchscreensnapshot 1`] = `
<LinearGradient
  colors={
    Array [
      "#6f347c",
      "#5d6fc4",
    ]
  }
  end={
    Object {
      "x": 1,
      "y": 0,
    }
  }
  start={
    Object {
      "x": 1,
      "y": 2,
    }
  }
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../app/assets/img/icon.png",
      }
    }
    style={
      Object {
        "resizeMode": "contain",
        "width": "80%",
      }
    }
  />
  <ActivityIndicator
    animating={true}
    color="#6f347c"
    hidesWhenStopped={true}
    size="large"
    style={
      Object {
        "flex": 1,
        "position": "absolute",
      }
    }
  />
</LinearGradient>
`;
